packages/react-native/src/primitives/PasswordField/PasswordField.tsx

```diff
@@ -24,12 +31,13 @@ export default function PasswordField({
       {...rest}
       disabled={disabled}
       secureTextEntry={obscureText}
+      style={[themedStyle.container, style]}
```

Comment:
Is this a `container` or the field itself?

Reply:
Currently, `TextField` applies the `style` prop to the container, not the field itself. ([link](https://github.com/aws-amplify/amplify-ui/blob/rna/main/packages/react-native/src/primitives/TextField/TextField.tsx#L39)). Perhaps we should have a separate `field` style for `PasswordField`?

Reply:
Yes I think so ([link](https://github.com/aws-amplify/amplify-ui/blob/rna/main/packages/react-native/src/primitives/TextField/TextField.tsx#L45))

Reply:
Got it. Definitely not within the scope of this PR but we may want to rethink that